packages/seed/src/fixture-storage.ts

```diff
@@ -115,7 +119,7 @@ export class FixtureStorage<T> {
    * @param uri Fixture uri
    */
   get(): T {
-    const fixture = pathExistsSync(this.path) && this.fs.read(this.path)
+    const fixture = this.exists && this.fs.read(this.path)
```

Comment:
this is okay, but would the `pathExistsSync(this.path)` suffice?

Reply:
yes, this will be changed in refactoring